app/controllers/campaigns_controller.rb

```diff
@@ -101,6 +101,14 @@ def remove_organizer
     redirect_to overview_campaign_path(@campaign.slug)
   end
 
+  def remove_course
+    campaigns_course = CampaignsCourses.find_by(course_id: params[:id], campaign_id: @campaign.id)
+    campaigns_course.destroy if campaigns_course
```

Comment:
If we aren't going to handle the case of failure — which will mainly happen when you try to remove a course that already got removed since the page was loaded — then it's better to let this raise a 500 instead of incorrectly flashing the 'removed' notice.

Alternatively, you could do something like this:
```ruby
message = campaign_course&.destroy ? 'campaign.course_removed' : 'campaign.course_already_removed'
flash[:notice] = t(message, …)
```